projects/dvl-fw/src/lib/plugins/ngx-dino/visualizations/scatterplot-visualization.ts

```diff
@@ -7,7 +7,10 @@ import { ScatterplotComponent } from '../components/scatterplot/scatterplot.comp
 
 
 export class ScatterplotVisualization extends DefaultVisualization {
-  readonly description = 'TODO';
+  // TODO remove below string
+  readonly description = `He determined to drop his litigation with the monastry, and relinguish
```

Comment:
Use lorem ipsum for the visualizations (for now). This site can generate varying amounts of text to test with:
https://loremipsum.io/#generator

Reply:
thanks for the url, i have taken text sample from this website.